server/controllers/product.js

```diff
@@ -0,0 +1,109 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const productModel = require('../model/product');
+const utils = require('../utils/utils');
+
+const app = express();
+
+const Logger = require('../services/logger_services');
+
+const logger = new Logger('app');
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+
+const createProduct = (req, res) => {
+  const id = utils.idGeneretor.generateId();
```

Comment:
The id is usually generated in the db, because you didn't have one I think this  utils.idGeneretor.generateId() need to be in the model layer. the controller didn't need to determine the id.